assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.stories.js

```diff
@@ -121,6 +121,51 @@ Ready.args = {
 	},
 };
 
+export const SearchConsoleGatheringData = Template.bind( {} );
+SearchConsoleGatheringData.storyName = 'Search Console Gathering Data';
+SearchConsoleGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport( [], {
+			options: searchConsoleArgs,
+		} );
+
+		for ( const options of analyticsArgs ) {
+			provideAnalyticsMockReport( registry, options );
+		}
+	},
+};
+
+export const AnalyticsGatheringData = Template.bind( {} );
+AnalyticsGatheringData.storyName = 'Analytics Gathering Data';
+AnalyticsGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		provideSearchConsoleMockReport( registry, searchConsoleArgs );
+
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
+
+export const AnalyticsAndSearchConsoleGatheringData = Template.bind( {} );
+AnalyticsAndSearchConsoleGatheringData.storyName =
+	'Analytics and Search Console Gathering Data';
+AnalyticsAndSearchConsoleGatheringData.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( MODULES_SEARCH_CONSOLE ).receiveGetReport( [], {
+			options: searchConsoleArgs,
+		} );
+
+		for ( const options of analyticsArgs ) {
+			registry
+				.dispatch( MODULES_ANALYTICS )
+				.receiveGetReport( [], { options } );
+		}
+	},
+};
```

Comment:
This story seems to be a duplicate of `DataUnavailable` below. The 'Data Unavailable' story is one that is a bit of a convention throughout so let's keep that one instead for this case.